src/avfoundation.cs

```diff
@@ -9164,15 +9200,19 @@ interface AVCapturePhotoSettings : NSCopying
 		[Internal]
 		[NoWatch, NoTV, NoMac, iOS (12, 0)]
 		[Export ("availableRawEmbeddedThumbnailPhotoCodecTypes")]
-		NSString[] _GetAvailableRawEmbeddedThumbnailPhotoCodecTypes { get; }
+		NSString[] _AvailableRawEmbeddedThumbnailPhotoCodecTypes { get; }
 
 		[NoWatch, NoTV, NoMac, iOS (12, 0)]
-		[Wrap ("Array.ConvertAll (_GetAvailableRawEmbeddedThumbnailPhotoCodecTypes, s => AVVideoCodecTypeExtensions.GetValue (s))")]
-		AVVideoCodecType[] GetAvailableRawEmbeddedThumbnailPhotoCodecTypes { get; }
+		[Wrap ("Array.ConvertAll (_AvailableRawEmbeddedThumbnailPhotoCodecTypes, s => AVVideoCodecTypeExtensions.GetValue (s))")]
+		AVVideoCodecType[] AvailableRawEmbeddedThumbnailPhotoCodecTypes { get; }
 
 		[NoWatch, NoTV, NoMac, iOS (12, 0)]
 		[NullAllowed, Export ("rawEmbeddedThumbnailPhotoFormat", ArgumentSemantic.Copy)]
-		NSDictionary RawEmbeddedThumbnailPhotoFormat { get; set; }
+		NSDictionary NativeRawEmbeddedThumbnailPhotoFormat { get; set; }
```

Comment:
note sure about the `Native` prefix...
the current one is `Weak` and, while it can be confusing we better select one that will fit everywhere (i.e. replace it once)

Reply:
Weak sounds good. And it is a pattern we use in other places.